C103

HadarRose opened the pull request:
Added Maven for dependency, removed local dependencies, removed other non-essential files

Commit:
Feedback implemented

Feedback regarding .gitignore and excluded files implemented. TODO: feedback about README.md; will be added with the end of this project

Commit:
Feedback implemented

Feedback regarding .gitignore and excluded files implemented. TODO: feedback about README.md; will be added with the end of this project

Commit:
Maven dependencies done

Commit:
Maven dependencies done

Commit:
Attempting to pull after rebase

Commit:
Small merge fixes

kev-chan commented on .gitignore:
```diff
@@ -1,10 +1,9 @@
-out/production/C102/C102_Runner.class
-src/C102_RunnerTest.java
-out/production/C102/C102_RunnerTest.class
+out/
```
please update the .gitignore with any needed files to ignore.
i.e. command line should only show files that actually need updating

kev-chan reviewed (changes requested):
wanted to see why the C102Runner.jar file was being checked in when you are running maven
also, does this work on command line?

Commit:
.gitignore modified

also inc

Commit:
Small change to .gitignore

Commit:
Bug fixes: properties filed compiled into JAR and is not ignored

-Fixed the compiled dependency on twitter4j.properties by deleting resource from pom.xml and by adding the file to the root directory
-Fixed twitter4j.properties not being ignored by removing it from repo and adding an example twitter4j_example.properties file instead
-Added instructions to README and updated them to according to the changes made

kev-chan commented on README.md:
```diff
@@ -3,10 +3,14 @@
  
 ### Instructions
+* Setup: In the root folder, create a *twitter4j.properties* that includes your credentials. You can make a copy of 
+*twitter4j_example.properties* in the same folder, and remove change the name to twitter4j.properties. 
```
instead of moving/ copy to file twitter4j.properties.  Moving it means you then get a git change for deletion which you don't actually want

kev-chan commented on pom.xml:
```diff
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
nice job with the dependency removals with the maven change

kev-chan commented on pom.xml:
```diff
@@ -0,0 +1,60 @@
+        <dependency>
+            <groupId>org.twitter4j</groupId>
+            <artifactId>twitter4j-core</artifactId>
+            <version>4.0.7</version>
```
let's move the versions to properties as it make it much easier to update in a single area.  It's also a pattern you will see in other code bases as well.

kev-chan commented on pom.xml:
```diff
@@ -0,0 +1,60 @@
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
same as above

Commit:
Dependencies versions moved to properties

also minor changes to readme file

kev-chan commented on README.md:
```diff
@@ -3,32 +3,19 @@
-### Using the Program
+### Instructions
+* Setup: In the root folder, create a *twitter4j.properties* that includes your credentials. You will need to make your 
+own file, but you can see twitter4j_example.properties, also in the root folder, for reference. Simply copy the content 
```
nice
